src/PPSDepartment/PPSDepartmentRequest.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Ietws;
+
+namespace ietws.PPSDepartment
+{
+
+    public class PPSDepartmentRequest : RequestBase
+    {
+        
+        public PPSDepartmentRequest(IetClient client) : base(client) { }
+
+        //curl -i -H "Accept: application/json" https://iet-ws.ucdavis.edu/api/iam/orginfo/pps/depts/search?deptCode=061419&key=myKey&v=1.0
+        public async Task<PPSDepartmentResults> Search(string value)
+        {
+
+            this.Url = "iam/orginfo/pps/depts/search";
+
+            this.QueryItems.Add("deptCode", value); //They can also search by a guid... but I don't care

```

Comment:
@jSylvestre how about adding a second arg for searchField that defaults to "deptCode" that way people can search by guid if they want to AND if they add other search options later you don't need to update the lib